server/src/services/ChannelService.ts

```diff
@@ -83,20 +86,31 @@ export const changeVisibilityWhenInvisible = (
     },
   });
 
+export const getChannelType = (
+  userId: string,
+  peerUserIds: string[],
+): ChannelType => {
+  if (peerUserIds.length === 1 && peerUserIds[0] === userId)
+    return ChannelType.self;
+
+  return ChannelType.private;
+};
```

Comment:
I intended this function to return ChannelType not boolean because there might be new feature added to server 'public' type channel. But now I know that in that case, we should create totally new different mutation function which is not `findOrCreatePrivateChannel`..

So I'll make change as you suggested, thanks always

Reply:
I had changed codes as you suggested  but it brings 3 test case failures in server side `Channel.test.ts`. I think it is because old `createNewChannel` function was used even when creating public channel.. 

So I had to revert that commit. 

I think we should refactor all related codes when we introduce `public` channel feature.

Reply:
I see. Thanks for trying 👍